packages/pinia/__tests__/store.spec.ts

```diff
@@ -379,4 +379,12 @@ describe('Store', () => {
       `[🍍]: A getter cannot have the same name as another state property. Rename one of them. Found with "anyName" in store "main".`
     ).toHaveBeenWarnedTimes(1)
   })
-})
+  it('warns when create store with exsist id', async () => {
```

Comment:
This seems like a typo, should be
```suggestion
  it('warns when creating store with existing id', async () => {
```

Reply:
Thank you, i change this text!